src/components/ButtonPanel.js

```diff
@@ -1,38 +1,46 @@
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] Also, remove this eslint disabled rule too, you want to avoid disabling a rule as essential as react-proptypes.

Reply:
Remove eslint diabled rule and fix elsint error